back/flag-service/src/flag/spec/FlagService.spec-integration.ts

```diff
@@ -236,6 +236,21 @@ describe('FlagService', () => {
         expect(pixel.author).toEqual('randomId');
         expect(pixel.hexColor).toEqual('#DDDDDD');
       });
+      it('return own pixel even when user did modify another pixel', async () => {
+        const myPixel = await flagService.addPixel(
+          'randomId',
```

Comment:
The fact that this id is random is irrelevant - however the fact that it's owned by the current user is

`thisUsersPixelId`

`anotherPixelId`

Reply:
(also modify L232+)